README.md

```diff
@@ -567,3 +568,341 @@ notebook
+  -k, --api-key TEXT              The API key to use to authenticate with
+                                  RStudio Connect.
+
+  -i, --insecure                  Disable TLS certification/host validation.
+  -c, --cacert FILENAME           The path to trusted TLS CA certificates.
+  --published                     Search only published content.
+  --unpublished                   Search only unpublished content.
+  --content-type [unknown|shiny|rmd-static|rmd-shiny|static|api|tensorflow-saved-model|jupyter-static|python-api|python-dash|python-streamlit|python-bokeh|python-fastapi]
+                                  Filter content results by content type.
+  --r-version VERSIONSEARCHFILTER
+                                  Filter content results by R version.
+  --py-version VERSIONSEARCHFILTER
+                                  Filter content results by Python version.
+  --title-contains TEXT           Filter content results by title.
+  --order-by [created|last_deployed]
+                                  Order content results.
+  -v, --verbose                   Print detailed messages.
+  --help                          Show this message and exit.
+
+$ rsconnect content search
+[
+  {
+    "max_conns_per_process": null,
+    "content_category": "",
+    "load_factor": null,
+    "cluster_name": "Local",
+    "description": "",
+    "bundle_id": "142",
+    "image_name": null,
+    "r_version": null,
+    "content_url": "https://connect.example.org:3939/content/4ffc819c-065c-420c-88eb-332db1133317/",
+    "connection_timeout": null,
+    "min_processes": null,
+    "last_deployed_time": "2021-12-02T18:09:11Z",
+    "name": "logs-api-python",
+    "title": "logs-api-python",
+    "created_time": "2021-07-19T19:17:32Z",
+    "read_timeout": null,
+    "guid": "4ffc819c-065c-420c-88eb-332db1133317",
+    "parameterized": false,
+    "run_as": null,
+    "py_version": "3.8.2",
+    "idle_timeout": null,
+    "app_role": "owner",
+    "access_type": "acl",
+    "app_mode": "python-api",
+    "init_timeout": null,
+    "id": "18",
+    "quarto_version": null,
+    "dashboard_url": "https://connect.example.org:3939/connect/#/apps/4ffc819c-065c-420c-88eb-332db1133317",
+    "run_as_current_user": false,
+    "owner_guid": "edf26318-0027-4d9d-bbbb-54703ebb1855",
+    "max_processes": null
+  },
+  ...
+]
+```
+
+See [this section](#searching-for-content) for more comprehensive usage examples
+of the available search flags.
+
+
+### Content Build
+
+> **Note:** The `rsconnect content build` subcommand requires RStudio Connect >= 2021.11.1
+
+RStudio Connect caches R and Python packages in the configured
+[`Server.DataDir`](https://docs.rstudio.com/connect/admin/appendix/configuration/#Server.DataDir).
+Sometimes, these package caches become stale and need to be rebuilt. This refresh
+occurs automatically when an RStudio Connect user visits the content via the
+RStudio Connect dashboard. Some content, however, is high priority or not accessed
```

Comment:
It will occur no matter how they visit the content (i.e. directly via vanity URL), right?

Reply:
Yes, that's right